examples/async_transcribe_local_media_file.js

```diff
@@ -15,23 +15,30 @@ const token = require('./config/config.json').access_token;
         metadata: "InternalOrderNumber=123456789",
         callback_url: "https://jsonplaceholder.typicode.com/posts",
         skip_diarization: false,
-        skip_punctuation: false,
-        speaker_channels_count: null,
-        custom_vocabularies: [{
+        skip_punctuation: false, //Optional value available with some languages
+        speaker_channels_count: null, //Optional value available with some languages
+        custom_vocabularies: [{  //Optional value available with some languages
             phrases: [
                 "add",
                 "custom",
                 "vocabularies",
                 "here"
             ]
         }],
-        filter_profanity: false,
-        remove_disfluencies: false,
-        delete_after_seconds: 2592000 // 30 days in seconds
+        filter_profanity: false, //Optional value available with some languages
+        remove_disfluencies: false, //Optional value available with some languages
+        delete_after_seconds: 2592000, // 30 days in seconds
+        language: "en" // ISO 639-1 2-letter language code. 
     };
 
     // Media may be submitted from a local file
-    var job = await client.submitJobLocalFile("./resources/example.mp3", jobOptions);
+    try {
+	    var job = await
```

Comment:
This won't work either because the job var is now scoped within the try block. The var needs to be declared outside the scope of the try block. I suggest the line above the try block.

```
var job;
try {
...
```